src/docs/docs.md

```diff
@@ -40,8 +40,13 @@ is derived from the docstrings of the classes implementing those symbols.
 Generating the site
 -------------------
 
+The site is generated with a script at `build-support/bin/publish_docs.sh`. To see the options available for this script, run:
+
+    :::bash
+    ./build-support/bin/publish_docs.sh -h
+
 To see http://www.pantsbuild.org/ site's content as it would be generated based on your local
-copy of the pants repo, enter the command
+copy of the pants repo, enter the command:
```

Comment:
Could you update the following code snippet to add a `-l` so that the preview is published locally, or remove the comment?

Reply:
I added a little section right under the code snippet describing the `-l` option -- could you let me know if that addresses your concern?